src/test/java/io/github/opendme/server/controller/CallControllerIT.java

```diff
@@ -125,6 +137,49 @@ void should_fails_on_vehicle_from_wrong_department() throws Exception {
         assertThat(response.getStatus()).isEqualTo(422);
     }
 
+    @Test
+    @WithMockUser
+    void should_create_call_response() throws Exception {
+        createCall();
+        createMember();
+
+        MockHttpServletResponse response = sendCreateResponseRequestWith(call.getId(), member.getId());
+
+        assertThat(response.getStatus()).isEqualTo(201);
+        List<CallResponse> allResponse = callResponseRepository.findAll();
+        assertThat(allResponse).hasSize(1);
+        var first = allResponse.getFirst();
+        assertThat(first.getMember().getId()).isEqualTo(member.getId());
+
+        var firstCall = first.getCall();
+
+        assertThat(firstCall).usingRecursiveComparison()
+                                     .comparingOnlyFields(
+                                             "id", "department", "vehicles"
```

Comment:
callResponses is empty in the expected result -> only compare the present fields. We can probably invert it and ignore the callResponses and createdAt fields instead.